src/contention_detectors/signal_analyzers/drop.hpp

```diff
@@ -131,6 +97,49 @@ class SignalDropAnalyzer : public SignalAnalyzer {
    */
   void shiftBasePoints();
 
+  //! int64_t
+  //! How far in the past we look.
```

Comment:
You could precise it a little bit: "Sets the number of previous iterations the algorithm takes in account"